src/lib/AnnotationControlsFSM.ts

```diff
@@ -21,18 +22,21 @@ export enum AnnotationInput {
 type Subscription = (state: AnnotationState) => void;
 
 export const modeStateMap = {
+    [AnnotationMode.DRAWING]: AnnotationState.DRAWING,
     [AnnotationMode.HIGHLIGHT]: AnnotationState.HIGHLIGHT,
     [AnnotationMode.NONE]: AnnotationState.NONE,
     [AnnotationMode.REGION]: AnnotationState.REGION,
 };
 
 export const modeTempStateMap = {
+    [AnnotationMode.DRAWING]: AnnotationState.NONE,
```

Comment:
Added this line because there was a type issue on L74 where because AnnotationMode has a `DRAWING` case, the type check thinks that we are missing a `[AnnotationMode.DRAWING]` case in the `modeTempStateMap` check (even though it is not possible for us to ever run into this case. Would be open to other suggestions as this is somewhat of a hack.